FRCore/FRCore/Network/RestClient.swift

```diff
@@ -33,6 +33,7 @@ public class RestClient: NSObject {
             config.httpCookieStorage = nil
             config.httpCookieAcceptPolicy = .never
             config.httpShouldSetCookies = false
+            config.urlCache = nil
```

Comment:
Maybe let's add some comments here on why we are doing this, same for the cookieStorage

Reply:
Added comments